stock_move_actual_date/security/stock_move_actual_date_security.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="group_actual_date_editable" model="res.groups">
+        <field name="name">Editable Actual Date</field>
```

Comment:
```suggestion
        <field name="name">Modify Actual Date</field>
```
This sounds better to me as a group name. "Editable Actual Date" sounds a bit misleading.